LICENSE.md

```diff
@@ -1,9 +1,21 @@
 MIT License
 
-Copyright (c) 2022 Stanford University
+Copyright (c) 2024, Paul Heidekrüger
```

Comment:
We normally point to the contributors file; e.g. spezi: https://github.com/StanfordSpezi/Spezi/blob/main/LICENSE.md 

The file headers should generally say:
[`SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)`](https://github.com/StanfordSpezi/Spezi/blob/cdafc915979573e4b82ab1dc0eac5a0444222ed6/.swiftlint.yml#L4)

Reply:
What licence info should I add for the TUM logo(s)?

Reply:
And keep the MIT licence, too?